app/Views/Templates/components/actions/dropdown/index.blade.php

```diff
@@ -0,0 +1,57 @@
+@props([
+    'variant' => 'regular', // Dropdown variant: regular or card
+    'contentRole' => 'primary', // Content role: primary, secondary, or tertiary
+    'position' => 'bottom', // Dropdown position: top, left, bottom, right
+    'align' => 'start', // Dropdown alignment: start or end
+    'label-text' => 'Dropdown', // Text for the dropdown button
```

Comment:
Did you check if it should be labelText and cardLabel vs label-text and card-label?